### PR TITLE
replace moment by moment-timezone

### LINKED ISSUE #1425: Atualização do moment e moment-timezone para horario de verão
### Contexto
Atualmente algumas aplicações do pagarme fazem uso do moment-timezone para forçar a utilização do timezone `America/Sao_Paulo`.
Para que o moment-timezone funcione corretamente precisamos utilizar a versão `0.5.26`.
Verificar se é necessário aplicar algo no cockpit

### Descrição técnica
- configurar a pilot para utilizar o moment-timezone versão `0.5.26`
- forçar a utilização do timezone  `America/Sao_Paulo`.

### PR DESCRIPTION
**Pull request by rsmelo**
## Contexto
Atualmente algumas aplicações do pagarme fazem uso do `moment-timezone` para forçar a utilização do timezone `America/Sao_Paulo`.
Para que o `moment-timezone` funcione corretamente precisamos utilizar a versão `0.5.26`.
Como o `moment-timezone` possui o `moment` contido nele a dependência do `moment` não é mais necessária. Sendo utilizada apenas como peer dependency para garantir a versão que precisamos.
na pratica foi necessario trocar todos os imports do pacote `moment` por `moment-timezone`.
Além de setar o timezone default como `America/Sao_Paulo`
https://github.com/pagarme/pilot/blob/3e385be1c28b1d9023ae097cadd858e091b4ec74/packages/pilot/src/initializeI18n.js#L10

### como testar?
- baixar a branch `bump/moment-timezone`
- instalar as novas dependencias `yarn start`
- navegar e aplicar filtros nas paginas que possuem datas
  - efetuar antecipação
  - efetuar saque
  - efetuar filtros em transações
  - efetuar filtros em extrato
  - efetuar filtros na home

## Issues linkadas
 Resolves #1425

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3e385be1c28b1d9023ae097cadd858e091b4ec74
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3e385be1c28b1d9023ae097cadd858e091b4ec74
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3e385be1c28b1d9023ae097cadd858e091b4ec74
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3e385be1c28b1d9023ae097cadd858e091b4ec74
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/cockpit/package.json [hunk 1]
```diff
@@ -21,12 +21,13 @@
     "jest": "24.8.0",
     "jest-cli": "24.8.0",
     "jest-diff": "24.8.0",
-    "moment": "2.24.0",
+    "moment-timezone": "0.5.27",
```
Comment (lucianopf) [inline packages/cockpit/package.json]: 👏  Ta ainda 1 minor maior do que a que usamos no core!

File: packages/pilot/src/initializeI18n.js [hunk 2]
```diff
@@ -3,10 +3,11 @@ import Backend from 'i18next-xhr-backend'
 import LanguageDetector from 'i18next-browser-languagedetector'
 import { reactI18nextModule } from 'react-i18next'
 
-import moment from 'moment'
+import moment from 'moment-timezone'
 import 'moment/locale/pt-br'
 
 moment.locale('pt-br')
+moment.tz.setDefault('America/Sao_Paulo')
```
Comment (lucianopf) [inline packages/pilot/src/initializeI18n.js]: Só lembrando que isso será configurado pra todo mundo que acessar a pilot, inclusive quem tiver em outra zona, por exemplo lá dos Estados Unidos 🤔

**Pull request merged**